File: Entities/SolutionReview.cs
using System;
using System.Collections.Generic;

namespace MindYourMoodWeb.Entities
{
    public class SolutionReview : Entity
    {
        //public int Id { get; set; }
        public ProblemIdea Idea { get; set; }
        public ICollection<SolutionPlan> SolutionSteps { get; set; }
        public string ReviewText { get; set; }
        public bool Achieved { get; set; }
        public DateTime AchievedDate { get; set; }
    }
}

File: ClientApp/src/app/_model/solutionReview.ts
import { ProblemIdea } from "./problemIdea";
import { SolutionPlan } from "./solutionPlan";

export interface SolutionReview {
  id: number;
  idea: ProblemIdea;
  solutionSteps: SolutionPlan[];
  reviewText: string;
  achieved: boolean;
  achievedDate: Date;
}
